packages/blaze/builtins.js

```diff
@@ -226,6 +226,11 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
         });
       },
       changedAt: function (id, newItem, oldItem, index) {
```

Comment:
So I could not reproduce that `changedAt` when data projected using `fields` didn't really change.

For example, if I have the following cursor over which I am doing `#each`:

```
Collection.find({}, {sort: {'value.a': 1}, fields: {'value.a': 1, test: 1}});
```

And values are inserted like:

```
Collection.insert({value: {a: 1}});
```
When I do:

```
Collection.update({}, {$set: {'value.b': 'b'}})
```

`changedAt` is not called at all.

And if I insert one one document in the middle, only Blaze for that one document is called, and no `changedAt` at all, for nothing.